src/pages/NotificationsPage.vue

```diff
@@ -184,8 +194,10 @@ export default {
     },
     getNotifications() {
       this.$serverGet(restApi.uri.sessionNotification).then((resp) => {
+        this.notis = []
         for (let i = 0; i < resp.data.length; i++) {
-          this.notis.push(resp.data[i])
+          const notification = resp.data[i]
+          this.notis.push(notification)
```

Comment:
Not a big deal but same thing?

Reply:
yes, this changed because I was initially filtering for unique notifications id and it made sense to have the notification as a variable